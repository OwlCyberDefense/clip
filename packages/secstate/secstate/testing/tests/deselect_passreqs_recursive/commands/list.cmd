/usr/sbin/secstate list password-req -a -r
