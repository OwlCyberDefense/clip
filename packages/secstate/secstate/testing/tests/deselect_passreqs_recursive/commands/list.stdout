[ ]Benchmark - ID: password-req, Title: 'Linux Password Requirements XCCDF', Profile: 'Custom'
	[ ]Profile - ID: profile-test, Title: 'Profile Test'
	[ ]Profile - ID: None, Title: 'None'
	[X]Profile - ID: Custom, Title: 'Customized profile from secstate'
	[ ]Group - ID: pass-reqs, Title: 'Linux Password Requirement Rules'
		[ ]Rule - ID: password-change-time, Title: 'GEN0000540 - Password Change Time'
		[ ]Rule - ID: passwd-accts-password, Title: 'GEN0000560 - Passwd Accounts Have Passwords'
		[ ]Rule - ID: pass-min-length, Title: 'GEN0000580 - Password Minimum Length'
		[ ]Rule - ID: pam-pass-min-length, Title: 'GEN0000580 - (PAM) Password Minimum Length'
		[ ]Rule - ID: pass-min-alpha-upper, Title: 'GEN0000600 - (PAM) Password Minimum Uppercase Characters'
		[ ]Rule - ID: pass-min-num, Title: 'GEN0000620 - (PAM) Password Minimum Numerics'
		[ ]Rule - ID: pass-min-special, Title: 'GEN0000640 - (PAM) Password Minimum Special Characters'
		[ ]Rule - ID: pass-no-repeat, Title: 'GEN0000680 - (PAM) Passwords Cannot Repeat Characters'
		[ ]Rule - ID: login-defs-pass-max-age, Title: 'GEN0000700-720 - (Login.defs) Passwords Are Changed Every N Days'
		[ ]Rule - ID: pass-max-age, Title: 'GEN0000700-720 - Passwords Are Changed Every N Days'
		[ ]Rule - ID: pass-min-reuse, Title: 'GEN0000800 - (PAM) Passwords Cannot Be Reused Within N Days'
